src/applications/personalization/profile/tests/components/Profile.unit.spec.js

```diff
@@ -268,14 +268,6 @@ describe('mapStateToProps', () => {
         expect(props.shouldShowDirectDeposit).to.be.false;
       });
     });
-    describe('when direct deposit info should not be fetched because user has not set up 2FA', () => {
```

Comment:
rather than delete this can it can be updated to be make sure it's `true` when the user is LOA3 but has not set up 2FA?